feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/PreviewViewModel.kt

```diff
@@ -110,14 +114,75 @@ class PreviewViewModel @AssistedInject constructor(
                                 currentCameraSettings = cameraAppSettings,
                                 systemConstraints = systemConstraints,
                                 zoomScale = zoomScale,
-                                previewMode = previewMode
+                                previewMode = previewMode,
+                                captureModeToggleUiState = getCaptureToggleUiState(
+                                    systemConstraints,
+                                    cameraAppSettings
+                                )
                             )
                     }
                 }
             }.collect {}
         }
     }
 
+    private fun getCaptureToggleUiState(
+        systemConstraints: SystemConstraints,
+        cameraAppSettings: CameraAppSettings
+    ): CaptureModeToggleUiState {
+        val cameraConstraints: CameraConstraints? = systemConstraints.forCurrentLens(
+            cameraAppSettings
+        )
+        val hdrDynamicRangeSupported = cameraConstraints?.let {
+            it.supportedDynamicRanges.size > 1
+        } ?: false
+        val hdrImageFormatSupported = cameraConstraints?.let {
+            it.supportedImageFormatsMap[CaptureMode.MULTI_STREAM]!!.size > 1
```

Comment:
Shouldn't this be `it.supportedImageFormatsMap[cameraAppSettings.captureMode]?.let { it.size > 1 }`?

SINGLE_STREAM mode will have only a single supported ImageCaptureFormat currently.